### PR TITLE
feature-javadoc

### PR DESCRIPTION
**Pull request by pablohincapie**
## Summary
I added documentation for the most important functions of the code include model and controller

## Trello Card
https://trello.com/c/KpYDwsud/47-javadoc

### CODE REVIEW

File: src/main/java/wolox/training/controllers/BookController.java [hunk 1]
```diff
@@ -34,37 +41,66 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
     @Autowired
     private BookRepository bookRepository;
 
+    /**
+     * Method that find all the information of one entity.
+     * @return
```
Comment (Gianni-Nicolas) [inline src/main/java/wolox/training/controllers/BookController.java]: Indicate what is returned after the @return tag. 
Apply this on every empty return

**Review by Gianni-Nicolas — CHANGES_REQUESTED** [review]

File: src/main/java/wolox/training/controllers/BookController.java (hunk 1 above)
Comment (pablohincapie) [reply src/main/java/wolox/training/controllers/BookController.java]: I added javadoc in tag retun

**Review by Gianni-Nicolas — DISMISSED** [review]

**Review by AleFalcon — DISMISSED** [review]
It's ok.

**Review by Gianni-Nicolas — APPROVED** [review]

**Review by AleFalcon — APPROVED** [review]
It's ok

**Pull request merged**